utils/build.ps1

```diff
@@ -478,6 +478,22 @@ function Get-BuildProjectCMakeModules([BuildComponent]$Project) {
   return "$BinaryCache\$($Project.value__)\cmake\modules"
 }
 
+function Get-TargetInfo($Arch) {
+  # Cache the result of "swift -print-target-info" as $Arch.targetInfo
+  $key = "targetInfo"
```

Comment:
Can we use the naming convention in the script and name the key `TargetInfo`? Additionally, I think that we should clarify that this is a cached value by introducing a `Cache` member in the `Arch` hashtable.